build.gradle

```diff
@@ -121,20 +122,11 @@ publishing {
             from components.java
             artifactId = 'hri-flink-pipeline-core'
         }
+
         mavenJava(MavenPublication) {
             artifact jarTests
         }
     }
-
-    repositories {
-        maven {
-            url 'TBD'
-            credentials {
-                username findProperty('user') ?: System.getenv('user')
-                password findProperty('password') ?: System.getenv('password')
```

Comment:
I thought this needed to be updated with the GitHub repo for publishing to GitHub packages? Is that not the case?

Reply:
> I thought this needed to be updated with the GitHub repo for publishing to GitHub packages? Is that not the case?

@alisakotliarova, I believe that is correct.

It should be something like:

```
               username = findProperty("GITHUB_ACTOR") ?: System.getenv("GITHUB_ACTOR")
               password = findProperty("GITHUB_TOKEN") ?: System.getenv("GITHUB_TOKEN")
```

See the validation/build.gradle here (on my PR-test branch) --> https://github.com/Alvearie/hri-flink-validation-fhir/tree/WHFHRI-699-test-pr/validation

Reply:
> > I thought this needed to be updated with the GitHub repo for publishing to GitHub packages? Is that not the case?
> 
> @alisakotliarova, I believe that is correct.
> 
> It should be something like:
> 
> ```
>                username = findProperty("GITHUB_ACTOR") ?: System.getenv("GITHUB_ACTOR")
>                password = findProperty("GITHUB_TOKEN") ?: System.getenv("GITHUB_TOKEN")
> ```
> 
> See the validation/build.gradle here (on my PR-test branch) --> https://github.com/Alvearie/hri-flink-validation-fhir/tree/WHFHRI-699-test-pr/validation

THis has now been Fixed. I am marking Resolved.

Reply:
Fixed now - I am marking resolved.